src/local-ssh/ipc/localssh.ts

```diff
@@ -81,10 +81,10 @@ export class LocalSSHServiceImpl implements LocalSSHServiceImplementation {
     }
 
     public async getWorkspaceAuthInfo(workspaceId: string) {
-        return retryWithStop(async (stop) => {
-            const getAuthInfo = async (id: string, client: Client<ExtensionServiceDefinition>) => {
+        return retryWithStop((stop) => {
+            const getAuthInfo = (id: string, client: Client<ExtensionServiceDefinition>) => {
                 try {
-                    return await client.getWorkspaceAuthInfo({ workspaceId });
+                    return client.getWorkspaceAuthInfo({ workspaceId });
```

Comment:
If we return the async function, try catch will not work, I will revert this change